sources/NSTextField+iTerm.m

```diff
@@ -38,4 +38,10 @@ - (int)separatorTolerantIntValue {
     }
 }
 
+- (NSUInteger)separatorTolerantUIntValue {
```

Comment:
Call it `UnsignedInteger` instead of `UInt` to hint that it's 64-bit (to match the `NSNumber` convention).